sssom/util.py

```diff
@@ -779,6 +779,10 @@ def to_mapping_set_dataframe(doc: MappingSetDocument) -> MappingSetDataFrame:
             m = get_dict_from_mapping(mapping)
             data.append(m)
     df = pd.DataFrame(data=data)
+    # The following 3 lines are to remove columns
+    # where all values are blank.
+    df = df.replace("", np.nan)
+    df = df.dropna(axis=1, how="all")
```

Comment:
I see you opted for this solution now. I think it is ok, let's roll with this for now. I can see some users complaining though when empty columns are indiscriminately dropped. Solving this cleanly may not be worth it right now, but just for documentation purposes, I will open a ticket.